RELEASING.adoc

```diff
@@ -3,12 +3,29 @@
 The actual release process is triggered by the release admin.
 In order to simplify that process, here is a list of prerequisites to be performed before asking the release admin to run the release scripts.
 
-1. Comment out the `spring-cloud-gcp-samples` module from the root `pom.xml` file, so that the samples aren't released.
+. Manually change the value of `TRACKING_HEADER_PROJECT_VERSION` in link:spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/UsageTrackingHeaderProvider.java[`UsageTrackingHeaderProvider`] to the version being released from the interim `BUILD-SNAPSHOT` value.
 
-2. Comment out any non-GA modules from the `pom.xml` file.
+. Change the value of the `<version>` elements of the `pom.xml` of all modules to the name of the version being released from the interim `BUILD-SNAPSHOT` values. This can be done using the https://www.mojohaus.org/versions-maven-plugin/set-mojo.html[mvn versions:set] command.
 
-3. Manually change the value of `TRACKING_HEADER_PROJECT_VERSION` in link:spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/UsageTrackingHeaderProvider.java[`UsageTrackingHeaderProvider`] to the version being released from the interim `BUILD-SNAPSHOT` value.
+    $ NEW_VERSION=[new-version]
+    $ ./mvnw versions:set -pl spring-cloud-gcp-dependencies -DnewVersion=$NEW_VERSION
```

Comment:
I don't know why, but the `spring-cloud-gcp-dependencies` module doesn't get processed by `versions:set` if run from the parent. So, we have to do it in a separate step.
Interestingly, `versions:revert` works fine for all modules when run from the parent.